superbuild/patches/eigen-3.3.8.patch

```diff
@@ -1,5 +1,14 @@
+From f69c4e8141acf38a3fccd42390713c8d92476be8 Mon Sep 17 00:00:00 2001
+From: Mathilde Merle <mathilde.merle@ihu-liryc.fr>
+Date: Mon, 1 Feb 2021 09:38:52 +0100
+Subject: [PATCH] eigen-3.3.8
+
+---
+ Eigen/src/Core/products/Parallelizer.h | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
 diff --git a/Eigen/src/Core/products/Parallelizer.h b/Eigen/src/Core/products/Parallelizer.h
```

Comment:
Maybe we should move to 3.3.9. The issue has been reported in https://gitlab.com/libeigen/eigen/-/issues/2011 and solved in 3.3.9.